### ISSUE TITLE
Request failed with status code 429

### ISSUE THREAD

**Issue by mariuskoeppen** [rizerphe/obsidian-companion#8]
After installing and inserting my API key and turning on completion I keep getting the error "Error Completing: Request failed with status code 429". 

Completion has not worked a single time. I tried switching models (ChatGPT, GPT-3 Davinci), still same error. 

What does this mean and how can it be solved?

**Comment by DimitrisMilonopoulos** [comment]
Maybe we need to have Plus for these models? Doesn't work for me either

**Comment by rizerphe** [comment]
No, this is not the matter of using Plus. OpenAI limits free accounts to three requests per minute. I'm thinking about what I can do to solve this, for now the easiest solution should be to add your credit card to your OpenAI account at https://platform.openai.com.

**Comment by rizerphe** [comment]
I added a "fallback" feature - if you have at least one preset saved, you can specify a "fallback" that'll be used to suggest completions while ChatGPT is rate limited. By specifying a different provider in this preset, you're able to still use completions, even though at a lower quality. The only way to keep the quality the same is unfortunately [upgrading your account](https://platform.openai.com/account/billing/overview). I'm closing this issue as there isn't much more I can do. If you have any further feedback, don't hesitate to let me know!

**Comment by mgsgde** [comment]
i am running in the rate limit error as well even though I have a paid subscription.
